molgenis-app/DOCKER.md

```diff
@@ -0,0 +1,53 @@
+# Run pull-requests and release candidates with Docker
+You can now functionally test pull-requests with docker / docker-compose.
+
+## Run only the MOLGENIS-image
+You can run only the image with your own local services
+
+```bash
+docker run registry.molgenis.org/molgenis/molgenis-app:#tag#
+```
+
+### On MacBook
+On mac you need to bind the service urls to you own host by adding *docker.for.mac.localhost*. 
+
+```bash
+docker run \
+    -e db_uri=jdbc:postgresql://docker.for.mac.localhost/molgenis \
+    -e elasticsearch.transport.addresses=docker.for.mac.localhost:9300 \
+    -e opencpu.uri.host=docker.for.mac.localhost \
+    #containerId#
+```
+
+## Run the MOLGENIS-image with integrated services
+We support 2 operating systems at the moment.
+- Windows 8 (and beyond)
+- OSX or MacOS
+
+### For Mac users
+With docker-compose:
+
+*Example*
+
+Run specific tag:
+
+```bash
+export TAG=PR-7492-3; docker-compose up
+```
+
+### For Windows users (from Windows 8 and forward)
+In powershell:
+
+*Example*:
+
+Run specific tag:
+
+```bash
+$env:TAG="PR-7492-3"; docker-compose up
```

Comment:
Running on my Windows 10 laptop:
```
> $env:TAG="PR-7492-3"; docker-compose up
Pulling molgenis (registry.molgenis.org/molgenis/molgenis-app:PR-7492-3)...
ERROR: Get https://registry.molgenis.org/v2/molgenis/molgenis-app/manifests/PR-7492-3: no basic auth credentials
```

Reply:
We'd put the docker registry behind a login. @sidohaakma suggests we open it up for anonymous users and will look at that today or tomorrow.

Reply:
Tried again resulting in the following error:
```
$env:TAG="PR-7492-3"; docker-compose up
Pulling molgenis (registry.molgenis.org/molgenis/molgenis-app:PR-7492-3)...
ERROR: manifest for registry.molgenis.org/molgenis/molgenis-app:PR-7492-3 not found
```